tests/utilities/test_seed.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import pytest
+
+import pytorch_lightning.utilities.seed as seed_utils
+
+
+def test_seed_stays_same_with_multiple_seed_everything_calls():
```

Comment:
shall we rather mock the 
```suggestion
@mock.patch.dict(os.environ, {})
def test_seed_stays_same_with_multiple_seed_everything_calls():
```